src/rwe/GameNetworkService.cpp

```diff
@@ -333,7 +333,7 @@ namespace rwe
         auto firstRelevantCommandIndex = (endpoint.nextCommandToReceive - firstCommandNumber).value;
 
         // if the packet is relevant (contains new information), process it
-        if (firstRelevantCommandIndex < static_cast<unsigned int>(message.command_set_size()))
+        if (firstRelevantCommandIndex < static_cast<int>(message.command_set_size()))
```

Comment:
`message.command_set_size()` is already `int`, could remove this cast altogether?